motor/foc_math.c

```diff
@@ -196,6 +196,10 @@ void foc_pll_run(float phase, float dt, float *phase_var,
 /**
  * @brief svm Space vector modulation. Magnitude must not be larger than sqrt(3)/2, or 0.866 to avoid overmodulation.
  *        See https://github.com/vedderb/bldc/pull/372#issuecomment-962499623 for a full description.
+ *        Null=V0 variation (reduces switching losses by up to 33%)
+ *        See the follwoing links for a full description: 
+ *        https://youtu.be/5eQyoVMz1dY
+ *        https://go.gale.com/ps/i.do?id=GALE%7CA18320578&sid=sitemap&v=2.1&it=r&p=AONE&sw=w&userGroupName=anon%7E469317a7
```

Comment:
```suggestion
 *        https://link.gale.com/apps/doc/A18320578/AONE
```
Seems to be the shorter link they like us to use in references.